backend/src/services/auth/auth-login-service.ts

```diff
@@ -258,7 +259,10 @@ export const authLoginServiceFactory = ({
     });
     // from password decrypt the private key
     if (password) {
-      const privateKey = await getUserPrivateKey(password, userEnc);
+      const privateKey = await getUserPrivateKey(password, userEnc).catch((err) => {
+        logger.error(err, `private key generation failed for user id - ${user.id} and email -${user.email}`);
```

Comment:
let's prefix with logs with the function they are being called from. this will help when we are querying

```
loginExchangeClientProof: private key generation failed for [userId=${user.id}] and [email=${user.email}] 
```

Reply:
Done